flow/Makefile

```diff
@@ -657,10 +659,42 @@ $(eval $(call do-copy,3_place,3_5_place_dp.odb,))
 
 $(eval $(call do-copy,3_place,2_floorplan.sdc,,.sdc))
 
+# This target does not produce a non-zero exit code if detailed or global
+# placement fails. In this case $(RESULTS_DIR)/3_place.odb and $(RESULTS_DIR)/3_place.sdc
+# are produced, such that an artifact can be created for inspection purposes.
+#
+# The check-place target subsequently will fail if
+# placement did not complete successfully, because $(RESULTS_DIR)/place.ok
+# is set to 0.
+#
+# This means that build systems, such as Bazel, that require a non-zero
+# exit code and the same artifacts to be produced every time, or it
+# will not publish the artifacts, can use this target.
```

Comment:
@QuantamHD how do your Bazel setup work?  Presumably you would have the same issue so I'm wondering if you solved it this way?

Reply:
I break my flow into steps (placement, cts, drt, etc.). Run them one at a time, and if a step fails I just rerun the tcl script for that stage which references outputs from the previous successful stage.

Reply:
I see.

My main use-cases for Bazel is that I want artifacts for projects that have hours of running time so that I can examine failed builds quickly. This allows me to turn around a tweak to my scripts in a short amount of time or run `make issue` so that I can easily send feature request/issues to the community(frequently using deltaDebug.py...)

This PR gives me artifacts for failed buidls for global place, detailed place, global route and detailed route.

Later, I suspect that I'll find it worth adding artifacts for failed builds to CTS.

Note! We are also using bazel_rules_hdl, but to create a Verilator artifact.